lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -130,6 +133,24 @@ def docs
 
     private
 
+    def self.run_puppet(command)
+      env_vars = self.puppet_bin.include?('/opt/puppetlabs') ? clean_env_vars : ::ENV
```

Comment:
I guess this is even more correct, right?

```suggestion
      env_vars = self.puppet_bin.start_with?('/opt/puppetlabs') ? clean_env_vars : ::ENV
```